index.html

```diff
@@ -13002,7 +13002,7 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				<p>Defines the number of items in the current set of listitems or treeitems. Not required if all elements in the set are present in the <abbr title="Document Object Model">DOM</abbr>. See related <pref>aria-posinset</pref>.</p>
 				<p>This <a>property</a> is marked on the members of a set, not the container element that collects the members of the set. To orient the user by saying an element is "item X out of Y," the <a>assistive technologies</a> would use X equal to the <pref>aria-posinset</pref> <a>attribute</a> and Y equal to the <code>aria-setsize</code> attribute.</p>
 				<p>If all items in a set are present in the document structure, it is not necessary to set this property, as the <a>user agent</a> can automatically calculate the set size and position for each item. However, if only a portion of the set is present in the document structure at a given moment (in order to reduce document size), this property is needed to provide an explicit indication of set size.</p>
-				<p>Authors MUST set the value of <code>aria-setsize</code> to an integer equal to the number of items in the set. If the total number of items is unknown, authors SHOULD set the value of <code>aria-setsize</code> to <code>-1</code>.</p>
+				<p>Authors MUST set the value of <code>aria-setsize</code> to an integer equal to the number of items in the set. If the total number of items is unknown, authors SHOULD set the value of <code>aria-setsize</code> to <code>-1</code>.
```

Comment:
This statement in setsize needs a conditional phrase similar to @mcking65's suggestion for posinset, above:
> Because it is not always required, the normative requirement needs a conditional phrase at the beginning. Otherwise, it can be read as requiring the attribute.

```suggestion
				<p>When specifying <code>aria-setsize</code>, authors MUST set the value to an integer equal to the number of items in the set. If the total number of items is unknown, authors SHOULD set the value of <code>aria-setsize</code> to <code>-1</code>.</p>
```